api/src/main/java/edu/cornell/mannlib/vitro/webapp/web/widgets/LoginWidget.java

```diff
@@ -198,38 +167,96 @@ private State getCurrentLoginState(HttpServletRequest request) {
             return State.LOGGED_IN;
         }
         if (isOutdatedLoginProcessBean(request)) {
-        	LoginProcessBean.removeBean(request);
+            LoginProcessBean.removeBean(request);
         }
         return LoginProcessBean.getBean(request).getState();
     }
 
-	/**
-	 * A LoginProcessBean is outdated unless the the "in-process" flag is set in the
-	 * session.
-	 *
-	 * Each time we hit Authenticate, the flag is set, and each time
-	 * we draw the widget it is reset.
-	 */
-	private boolean isOutdatedLoginProcessBean(HttpServletRequest request) {
-		boolean inProcess = LoginInProcessFlag.checkAndReset(request);
-		if (!inProcess) {
-			log.debug("The process bean is outdated. Discard it.");
-		}
-
-		return !inProcess;
-	}
-
-	/** What's the URL for this servlet? */
+    /**
+     * A LoginProcessBean is outdated unless the the "in-process" flag is set in the
+     * session.
+     * <p>
```

Comment:
should this <p> tag be here?

Reply:
id does not have to be here, it is there to indicate a blank line